content/sdk/80.troubleshooting.md

```diff
@@ -8,61 +8,67 @@ navigation: false
 Whether you’re facing integration challenges, encountering unexpected errors, or need clarity on certain functionalities,
 here are detailed solutions and best practices to ensure a smooth development experience.
 
-::card-group
-::card
----
-title: JavaScript
-icon: i-heroicons-code-bracket
-to: #
----
-Troubleshooting JavaScript SDK Issues.
-::
-::card
----
-title: GO
-icon: i-heroicons-cog
-to: #
----
-Troubleshooting Go SDK Issues.
-::
-::card
----
-title: Python
-icon: i-heroicons-circle-stack
-to: #
----
-Troubleshooting Python SDK Issues.
-::
-::card
----
-title: Java
-icon: i-heroicons-rectangle-group
-to: #
----
-Troubleshooting Java SDK Issues.
-::
-::card
----
-title: Swift
-icon: i-heroicons-rocket-launch
-to: #
----
-Troubleshooting Swift SDK Issues.
-::
-::card
----
-title: Rust
-icon:  i-heroicons-wrench
-to: #
----
-Troubleshooting Rust SDK Issues.
-::
-::card
----
-title: Bun
-icon: i-heroicons-cube
-to: #
----
-Troubleshooting Bun Issues.
-::
-::
+## JavaScript SDK
+
+  ::collapsible
+    :field If you encounter issues using Js SDK during the installation, refer to the troubleshooting steps:
+
+If you encounter issues while using the zkSync-era Javascript V5 SDK, please refer to the following troubleshooting tips:
+
+1. **Network Selection**: Ensure you are using the Layer 2 (L2) network instead of Layer 1 (L1) Sepolia. The correct
+endpoint for the
+L2 network is: [zkSync Sepolia Explorer](https://sepolia.explorer.zksync.io/).
```

Comment:
We have tags that make referring to common values like the zkSync sepolia explorer easier to manage. Replace the URL with `%%zk_testnet_block_explorer_url%%`